guides/snp.md

```diff
@@ -0,0 +1,108 @@
+# SNP Guide
+
+## Platform Setup
+
+In order to launch SEV or SNP memory encrypted guests, the host must be prepared with a compatible kernel. Required changes in external components are still under development but are planned to be upstream soon.

```

Comment:
We should almost certainly have an explicit reference to the version of the host kernel that we expect today (i.e. the version used in the CI).

Just say that CoCo main expects this host kernel which you can get from wherever and we will upgrade to the upstream host kernel soon.